z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -386,24 +387,26 @@ After above steps, you will get one ready OpenShift Container Platform on the IB
 ## Day2 Operation
 
 ### Add compute node
-Use this playbook to add a new compute node as allocated IP:
+Use this playbook to add a new compute node as allocated IP and update bastion info automatically:
 ```sh
 ansible-playbook -i inventory.yaml add-new-compute-node.yaml 
 ```
-Use this playbook to add a new compute node as fixed IP:
+Use this playbook to add a new compute node as fixed IP and update bastion info automatically:
 ```sh
 ansible-playbook -i inventory.yaml add-new-compute-node.yaml -e ip=x.x.x.x
 ```
 Use this playbook to add multiple compute nodes as allocated IP, and update bastion info automatically:
 ```sh
-ansible-playbook -i inventory.yaml add-new-compute-node.yaml -e worker_number=3 -e update_bastion=true
+ansible-playbook -i inventory.yaml add-new-compute-node.yaml -e worker_number=3 
 ```
 Use this playbook to add multiple compute nodes as fixed IP, separate the IP list with commas, and update bastion info automatically:
 ```sh
-ansible-playbook -i inventory.yaml add-new-compute-node.yaml -e ip=x.x.x.x,x.x.x.x -e worker_number=2 -e update_bastion=true
+ansible-playbook -i inventory.yaml add-new-compute-node.yaml -e ip=x.x.x.x,x.x.x.x -e worker_number=2
 ```
 **Please notice:**
-> If you use your own bastion server, you can refer [Add-DNS-HAProxy](docs/add-dns-haproxy.md) to update bastion info.
+> We will default treat bastion server and localhost server as a same one.
```

Comment:
@shhull , I add more information here with internal bastion.

Reply:
We can remove the ` and update bastion info automatically` text in adding compute node part, because we update bastion based on the `use_internal_bastion` value from inventory file, which the user already knows.

Reply:
Also we need remove bation.yaml step